db/migrate/20191106095154_populate_document_type_id_in_metadata_revisions.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class PopulateDocumentTypeIdInMetadataRevisions < ActiveRecord::Migration[5.2]
+  def change
+    type_doc_ids_hash = Document.group(:document_type_id).pluck(:document_type_id, "ARRAY_AGG(id)").to_h
+
+    type_doc_ids_hash.each { |type, ids|
+      MetadataRevision.joins("INNER JOIN revisions ON metadata_revisions.id = revisions.id")
+        .where("revisions.document_id": ids).update_all(document_type_id: type)
+    }
```

Comment:
Minor style thing that, it's conventional for multiline blocks to use `do` and `end` - whereas single line blocks should use the curly braces: https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/BlockDelimiters